### DISCUSSION TITLE
[Feature Request] Add Subject to Body

### DISCUSSION THREAD

**Discussion by ldpg-dev** [anonaddy/anonaddy#365] [Feature Requests]
Would it be possible to have an option to add the email subject to the top of the body before encrypting and then another one to strip the subject out of the forwarded email completely?

Perhaps the options can look something like: 
* Remove Email Subject (True/False)
* Append Email Subject in Body

This can be useful as subjects aren't really encrypted. Removing the email subject is a nice to have but at the same time, I could just add a dot to the Replace Email Subject option, but some times I do want to see what the subject says, that's why having the subject within the encrypted body can be so useful.

**Comment by willbrowningme** [comment]
This already happens if you set a subject in the [settings page](https://app.anonaddy.com/settings) under "Replace Email Subject". It is added to the email banner information. Now if you have the banner information turned off you won't see this. I'll try to think of a better more flexible solution.

**Comment by ldpg-dev** [comment]
Thanks for getting back. This solution works for me, thank you!

**Comment by ldpg-dev** [comment]
@willbrowningme Would it be possible for me as the user to edit the content of the Email Banner from the settings menu? That might be a flexible solution that could be easy to implement.

**Comment by willbrowningme** [comment]
@BethelConsultingDev yes I do like that idea, I'll see if I can add placeholders e.g. {subject} etc. so you can write your own.

**Comment by ldpg-dev** [comment]
@willbrowningme You're the best!
